yarn.lock

```diff
@@ -2231,6 +2231,20 @@
     classnames "^2.3.1"
     motion-ui cultureamp/motion-ui
 
+"@kaizen/component-library@^14.3.4":
```

Comment:
I saw this too. I've traced this change down to an RTE dependency, which I guess the `yarn.lock` was forgotten about.

Reply:
Haha up to you. I'm not too fussed if it's still in here. We're also in a race, since it's in both our PRs 😜